### PR TITLE
fix swiping issue and pagination indicators

### PR DESCRIPTION
**Pull request by PoorTable**
Issue described here: https://github.com/kidjp85/react-id-swiper/issues/466 
also fixed: 
- swiping on mobile (for some reason for me it was broken)
- now we displaying one item per time

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "prismjs": "^1.29.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "swiper": "^5.4.1",
```
Comment (sharnik) [inline package.json:46]: It doesn't seem that this dependency is used, why do we need that?

**Review by sharnik — COMMENTED** [review]
On the mobile version the team member slides got much bigger. We should stick to the current size just don't show the prev/next slide previews.

CleanShot 2023-05-04 at 08 30 53@2x

**Comment by PoorTable** [comment]
> On the mobile version the team member slides got much bigger. We should stick to the current size just don't show the prev/next slide previews.
> 
> CleanShot 2023-05-04 at 08 30 53@2x

Hm, It wasn't me directly, but i'll try to find what causing it

File: package.json (hunk 1 above)
Comment (PoorTable) [reply package.json:46]: `react-id-swiper` is a wrapper lib for a `swiper` and as I understood from issues and documentation the `swiper` dependency is needed for some features to work, without this dependency dots won’t work, You can try it locally

**Comment by sharnik** [comment]
> It wasn't me directly, but i'll try to find what causing it

It was. When you removed `slidesPerView={1.4}` it allowed more width for the slides to grow. Instead, we should adjust margin/padding to fill it with whitespace.

**Review by sharnik — APPROVED** [review]
Looks good!

**Pull request merged**